### PR TITLE
copy fixes

### PR DESCRIPTION
**Pull request by diya2214**

### CODE REVIEW

File: app/config/question-bank.js [hunk 1]
```diff
@@ -1946,11 +1946,11 @@ const questionBank = {
           },
           hint: {
             html: `
-            Enter the estimated surface area of the replacement, new or expanded store
+            Enter the estimated cover surface area of the existing store
             <br/><br/>
             Enter size in metres squared
             <br/><br/>
-            <p class='govuk-body'>Existing store</p>
+            <p class='govuk-body'><b>Existing store</p>
```
Comment (zavies97) [inline app/config/question-bank.js:1953]: do you need to close the <b> tag after Existing store?

Comment (zavies97) [reply app/config/question-bank.js:1953]: < b > tag

File: app/config/question-bank.js [hunk 2]
```diff
@@ -2239,7 +2239,7 @@ const questionBank = {
           pageTitle: '',
           title: 'What type of slurry separator will you have?',
           hint: {
-            text: 'Capable to processing at least 3m3 of slurry per hour and producing at least 25% dry matter. '
+            text: 'Capable to processing at least 3m³ of slurry per hour and producing at least 25% dry matter. '
```
Comment (zavies97) [inline app/config/question-bank.js:2242]: This looks good based on ticket requirements, but isn't really good English

Nothing to change here, just wanted to raise it :)

Comment (diya2214) [reply app/config/question-bank.js:2245]: updated as per US

File: app/config/question-bank.js (hunk 1 above)
Comment (diya2214) [reply app/config/question-bank.js]: fixed

File: test/integration/narrow/helpers/other-items-sizes.test.js [hunk 3]
```diff
@@ -288,8 +288,8 @@ describe('Other Items Sizes Array Function', () => {
           {
             type: 'MIN_MAX',
             min: 1,
-            max: 9999,
-            error: 'Quantity must be between 1-9999'
+            max: 999999,
+            error: 'Quantity must be between 1-999999'
```
Comment (zavies97) [inline test/integration/narrow/helpers/other-items-sizes.test.js:292]: image


Pumps should be 1-9999

File: test/integration/narrow/helpers/other-items-sizes.test.js [hunk 4]
```diff
@@ -325,8 +325,8 @@ describe('Other Items Sizes Array Function', () => {
...
```
Comment (zavies97) [inline test/integration/narrow/helpers/other-items-sizes.test.js:329]: image
Pumps should be 1-9999

File: test/integration/narrow/helpers/other-items-sizes.test.js [hunk 5]
```diff
@@ -399,8 +399,8 @@ describe('Other Items Sizes Array Function', () => {
...
```
Comment (zavies97) [inline test/integration/narrow/helpers/other-items-sizes.test.js:403]: image
Pumps should be 1-9999

File: test/integration/narrow/helpers/other-items-sizes.test.js [hunk 6]
```diff
@@ -621,8 +621,8 @@ describe('Other Items Sizes Array Function', () => {
...
```
Comment (zavies97) [inline test/integration/narrow/helpers/other-items-sizes.test.js:625]: image

agitators and aerators should be 1-9999

File: test/integration/narrow/helpers/other-items-sizes.test.js [hunk 7]
```diff
@@ -658,8 +658,8 @@ describe('Other Items Sizes Array Function', () => {
...
```
Comment (zavies97) [inline test/integration/narrow/helpers/other-items-sizes.test.js:662]: image
Agitators should be 1-9999

File: test/integration/narrow/helpers/other-items-sizes.test.js [hunk 8]
```diff
@@ -695,8 +695,8 @@ describe('Other Items Sizes Array Function', () => {
...
```
Comment (zavies97) [inline test/integration/narrow/helpers/other-items-sizes.test.js:699]: image
Inspection platform (this item) should be 1-9999 (look at safety equipment number 1

File: test/integration/narrow/helpers/other-items-sizes.test.js [hunk 9]
```diff
@@ -302,7 +302,7 @@ describe('Other Items Sizes Array Function', () => {
         hint: {
           text: 'Grant amount: £2,090 per pump'
         },
-        classes: 'govuk-input--width-3',
+        classes: 'govuk-input--width-4',
```
Comment (zavies97) [inline test/integration/narrow/helpers/other-items-sizes.test.js:305]: Pretty sure this is from round 1 and should therefore be left as a 3

**Review by zavies97 — CHANGES_REQUESTED** [review]

File: app/config/question-bank.js [hunk 10]
```diff
@@ -1865,6 +1865,9 @@ const questionBank = {
             Enter the estimated surface area of the replacement, new or expanded store
             <br/><br/>
             Enter size in metres squared
+            <br/><br/>
+            <p class="govuk-body"><b>Grant-funded store</p>
```
Comment (zavies97) [inline app/config/question-bank.js:1869]: needs a </ b > tag

File: app/config/question-bank.js [hunk 11]
```diff
@@ -2035,7 +2038,7 @@ const questionBank = {
               type: 'number',
               classes: 'govuk-input--width-5',
               label: {
-                html: '<p class="govuk-body">Grant-funded store</p>',
+                html: '<p class="govuk-body"><b>Grant-funded store</p>',
```
Comment (zavies97) [inline app/config/question-bank.js:2041]: needs a </ b > tag

File: app/config/question-bank.js [hunk 12]
```diff
@@ -2372,19 +2375,19 @@ const questionBank = {
...
               {
                 linkedPrefix: 'Size: ',
-                linkedSufix: 'm²',
-              },
+                linkedSufix: 'm²'
+              }
             ],
             linkedQuestionyarkey: ['serviceCapacityIncrease', 'coverSize', 'existingCoverSize'],
-            dependentQuestionKeys: ['storageType', 'coverType', 'existingCoverType', 'separatorOptions'],
+            dependentQuestionKeys: ['storageType', 'coverType', 'existingCoverType', 'separatorType']
```
Comment (zavies97) [inline app/config/question-bank.js:2390]: leave this as separatorOptions pls. I need to add all the selected separator details to a unique yarkey for the sidebars/project summary/email/dora so please leave this as is

Comment (diya2214) [reply app/config/question-bank.js:2390]: its a bug raised by Rev it was there just showing the wrong heading

File: test/integration/narrow/helpers/other-items-sizes.test.js [hunk 13]
```diff
@@ -732,8 +732,8 @@ describe('Other Items Sizes Array Function', () => {
           {
             type: 'MIN_MAX',
             min: 1,
-            max: 999999,
-            error: 'Quantity must be between 1-999999'
+            max: 9999,
+            error: 'Quantity must be between 1-9999'
```
[the remark this answers is not in the document]
Comment (zavies97) [reply test/integration/narrow/helpers/other-items-sizes.test.js:736]: image
image

File: app/config/question-bank.js (hunk 12 above)
Comment (zavies97) [reply app/config/question-bank.js:2390]: If its a header issue, that should be sorted in models.js (See line 33-50)

Comment (zavies97) [reply app/config/question-bank.js:2390]: I found and fixed the issue (moels.js switch case didn't have a break for separator. Its fixed now with separatorOptions

**Review by zavies97 — APPROVED** [review]

**Review by yunusfirat — APPROVED** [review]

**Pull request merged**